tests/integ_tests/workflow_tests/activities.rs

```diff
@@ -793,3 +795,15 @@ async fn one_activity_abandon_cancelled_after_complete() {
         .unwrap();
     assert_matches!(res, WorkflowExecutionResult::Succeeded(_));
 }
+
+#[tokio::test]
+async fn it_can_complete_async() {
+    let mut starter = CoreWfStarter::new("bah");
+    let mut worker = starter.worker().await;
+
+    async fn activity(_: ActContext, _: String) -> anyhow::Result<ActExitValue<()>> {
+        Ok(ActExitValue::WillCompleteAsync)
+    }
+
+    worker.register_activity("async_activity", activity);
```

Comment:
Test should get fleshed out with a workflow that runs this, and then complete the activity with a client.

Reply:
done